jenkins/custom-ci/generic/custom-ci.groovy

```diff
@@ -439,7 +439,7 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
                 script {
                     try {
-                        def build_cortx_all_image = build job: 'cortx-all-docker-image', wait: true,
+                        def build_cortx_all_image = build job: 'cortx-image', wait: true,
```

Comment:
I guess this change is not intended for CORTX-25630 
Also, we need to trigger - https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/cortx-all-docker-image/

Reply:
@shailesh-vaidya, Yes. removed it.